src/features/campaigns/models/CampaignActivitiesModel.ts

```diff
@@ -260,9 +260,9 @@ export default class CampaignActivitiesModel extends ModelBase {
       return new ErrorFuture(activities.error);
     }
 
-    const now = new Date();
+    const nowDate = new getUTCDateWithoutTime(new Date());
```

Comment:
and apply same logic as above = create a Date object from the string value returned by `getUTCDateWithoutTime()`!